<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDAProjectDetailDialog</class>
 <widget class="QDialog" name="QDAProjectDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="viewProjectConfigText">
     <property name="plainText">
      <string>project config
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="createProjectBtn">
     <property name="text">
      <string>Create Project (创建项目)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="loadProjectConfigBtn">
     <property name="text">
      <string>Load Project Config (加载项目配置)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="saveProjectConfigBtn">
     <property name="text">
      <string>Save Project Config (保存项目配置)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
